cluster-autoscaler/config/const.go

```diff
@@ -17,6 +17,10 @@ limitations under the License.
 package config
 
 const (
+	// SchedulerConfigFileFlag is the name of the flag
+	// for passing in custom scheduler config for in-tree scheduelr plugins
+	SchedulerConfigFileFlag = "scheduler-config-file"
```

Comment:
So that it is easier to refer again [here](https://github.com/kubernetes/autoscaler/pull/5708/files/7ba7fdc1009138ac29d8d894015a8c0e2ff17ca1#diff-76f2867167fb87a2c61e82ea733b893d48724ca43e861b86a96f6dc582805ad4R261).